service/history/service.go

```diff
@@ -114,7 +114,13 @@ func (s *Service) Start() {
 	})
 
 	rawHandler := handler.NewHandler(s.Resource, s.config, wfIDCache)
```

Comment:
Will the rate limited handler contain other limit enforcements that are currently in handler? Just making sure we are on the same page because that was my understanding based on [this discussion](https://github.com/uber/cadence/pull/5684#discussion_r1500922598).

Reply:
Yes, I have tasks to move the ratelimiter to this wrapper, as well as the metrics to their own wrapper.